testgrid/config.yaml

```diff
@@ -2974,6 +2985,20 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: 'davanum+testgrid@gmail.com'
 
+# Alibaba Cloud Provider Dashboard
+- name: conformance-alibaba-cloud-provider
+  dashboard_tab:
+  - name: "Alibaba Cloud Provider, v1.10"
+    description: "Runs conformance tests for cloud provier alibaba cloud on release v1.10"
```

Comment:
just to check, this is the stable release of 1.10 right? (versus checking out and building from the branch, or a tag)

Reply:
This is the stable release of v1.10.4 from official repo.  Not build from branch.